prov/rxm/src/rxm.h

```diff
@@ -280,18 +281,26 @@ struct rxm_mr {
 	struct rxm_domain *domain;
 };
 
-struct rxm_ep_wire_proto {
-	uint8_t	ctrl_version;
-	uint8_t	op_version;
-	uint8_t endianness;
-	uint8_t padding[5];
-	uint64_t eager_size;
-};
-
-struct rxm_cm_data {
-	struct sockaddr name;
-	uint64_t conn_id;
-	struct rxm_ep_wire_proto proto;
+union rxm_cm_data {
+	/* connect */
+	struct {
+		uint8_t version;
+		uint8_t endianness;
+		uint8_t ctrl_version;
+		uint8_t op_version;
+		uint16_t port;
+		uint8_t padding[2];
+		uint64_t eager_size;
+		uint64_t client_conn_id;
+	};
```

Comment:
Define this as struct _connect { .. } connect.

Reply:
I can add the connect variable but do we also need to name the struct?

Reply:
See https://github.com/ofiwg/libfabric/pull/4814/files.  Based on that and other PRs, I'm guessing someone is building this using a C++ compiler (through their own build system).